moped-editor/src/views/projects/projectView/ProjectMilestones.js

```diff
@@ -92,8 +94,8 @@ const useColumns = ({
         field: "moped_milestone_related_phase",
         editable: true, // this is to be able to use the renderEditCell option to update the related phase
         // during editing -- the input field is always disabled
-        renderCell: (props) =>
-          phaseNameLookup[props.row.moped_milestone?.related_phase_id] ?? "",
+        valueGetter: (value, row) =>
+          phaseNameLookup[row.moped_milestone?.related_phase_id] ?? "",
```

Comment:
bc of the fact that this is a view only field and the `ViewOnlyTextField` below is based off the value of another field, `props.row.moped_milestone`, i was able to just update this to `valueGetter` which actually sets the value for the field and so sorting will just work without having to add a custom sort function